### PR TITLE
[perceval] Update license and copyright information

### PR DESCRIPTION
**Pull request by vchrombie**
This PR updates license information for every source code file in Perceval.

I tried updating this using the tool, [vchrombie/grimoirelab-scripts](https://github.com/vchrombie/grimoirelab-scripts).

### CODE REVIEW

**Comment by vchrombie** [comment]
Hi @valeriocos 

Ok, I will look at why the tests are failing. Meanwhile, I need some review of this draft PR.

**Comment by vchrombie** [comment]
There are some unexpected conditions with a few files where it deletes some part of code too. I will dig a bit and see what is the problem.

**Comment by valeriocos** [comment]
@vchrombie thank you for the PR! perfect, thanks!

File: perceval/backends/core/discourse.py [hunk 1]
```diff
@@ -16,10 +16,12 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 # Authors:
-#    Santiago Dueñas <sduenas@bitergia.com>
-#    J. Manrique López de la Fuente <jsmanrique@bitergia.com>
```
Comment (vchrombie) [inline perceval/backends/core/discourse.py]: I checked the [commit history of discourse.py](https://github.com/chaoss/grimoirelab-perceval/commits/master/perceval/backends/core/discourse.py), but couldn't find any reference of jsmanrique as an author.

Comment (valeriocos) [reply perceval/backends/core/discourse.py]: I checked the history too and I don't see him. The change that introduced it is https://github.com/chaoss/grimoirelab-perceval/commit/d36293a4ba3cd4e6c95be900c145c09bf57c6f00.

I would leave the name there, probably he helped in writing the file, but it was committed by someone else.

Comment (vchrombie) [reply perceval/backends/core/discourse.py]: Thanks for the reply. I will revert the changes.

Comment (valeriocos) [reply perceval/backends/core/discourse.py]: thanks! (I hope this isn't too much work)

Comment (vchrombie) [reply perceval/backends/core/discourse.py]: > thanks! (I hope this isn't too much work)

Not really but, I am worried that the script won't work well in such corner cases.

**Comment by vchrombie** [comment]
Taking this [comment](https://github.com/chaoss/grimoirelab-perceval/pull/623#discussion_r389359149) into consideration, I am reverting a few deleted author names and update the PR.

File: perceval/backends/core/discourse.py (hunk 1 above)
Comment (valeriocos) [reply perceval/backends/core/discourse.py]: I see, I'd rely on the script and see how many corner cases pop up.

**Comment by valeriocos** [comment]
Please @vchrombie ping me when the PR is ready for review. Consider to add some extra info to the commit description (saying that the script updated also the list of authors), thanks! :)

**Comment by vchrombie** [comment]
> Please @vchrombie ping me when the PR is ready for review. Consider to add some extra info to the commit description (saying that the script updated also the list of authors), thanks! :)

Sure @valeriocos. :slightly_smiling_face: 
Thanks for the suggestions.

**Comment by valeriocos** [comment]
you're welcome!

File: perceval/backends/core/mbox.py [hunk 2]
```diff
@@ -18,9 +18,14 @@
 # Authors:
 #     Santiago Dueñas <sduenas@bitergia.com>
 #     Germán Poo-Caamaño <gpoo@gnome.org>
+#     Stephan Barth <stephan.barth@gmail.com>
+#     valerio <valerio.cosentino@gmail.com>
+#     Valerio Cosentino <valcos@bitergia.com>
```
Comment (vchrombie) [inline perceval/backends/core/mbox.py]: Hi @valeriocos 

Just a small doubt, do these both belong to you? :sweat_smile: 
If yes, do I need to merge them or leave it like this?

Comment (valeriocos) [reply perceval/backends/core/mbox.py]: eheh thanks! yes, it belongs to me, can you merge them?

**Comment by vchrombie** [comment]
Hi @valeriocos 

I think I am done. Can you have a look when you are free?

**Comment by vchrombie** [comment]
> Minor comment: you should appear as an author of thes files, since this commit will modify them. 

Makes sense.

> * the next time the script is executed, you will be listed as an author

In how time period you think it would be better to run this script?

> In a next iteration, you can try to target also the other files not targeted in this PR

Did I miss any files? I have covered all the files in the perceval folder.

**Comment by valeriocos** [comment]
sorry @vchrombie I have just realized the the commit message is not properly rendered in the terminal. Everything appears in a single line (see below)
captura_240

Can you check this? Thanks

**Comment by vchrombie** [comment]
> https://github.com/chaoss/grimoirelab-perceval/pull/623#issuecomment-596216564

Sure, let me fix it right now.

**Comment by valeriocos** [comment]
> In how time period you think it would be better to run this script?

It depends on the repo activity. A monthly-based execution could be good, or every 2/4/6 months seems also reasonable. WDYT?

> Did I miss any files? I have covered all the files in the perceval folder.

I was referring to the other folders (tests, bin). Maybe the script could be updated to target all files (or specific directories), WDYT?

**Comment by valeriocos** [comment]
> Sure, let me fix it right now.

Thanks, it looks better now!

**Comment by vchrombie** [comment]
> It depends on the repo activity. A monthly-based execution could be good, or every 2/4/6 months seems also reasonable. WDYT?

I was thinking on a yearly basis. :sweat_smile: 
I think 6 months would be good enough.
 
> I was referring to the other folders (tests, bin). Maybe the script could be updated to target all files (or specific directories), WDYT?

Ah, my bad again. I forgot about those files. I was concentrating on just perceval folder.
If you give me 5 min, I can get you that done too. WDYT?

**Comment by valeriocos** [comment]
oops, already merged! No worries, you can submit another PR targeting the remaining folders, thanks!

**Pull request closed without merging**